less/circle-background.less

```diff
@@ -0,0 +1,164 @@
+@circle-bg-xs-diameter: 25%;
+@circle-bg-sm-diameter: 50%;
+@circle-bg-md-diameter: 120%;
+@circle-bg-lg-diameter: 200%;
+@circle-bg-xl-diameter: 400%;
+@circle-bg-translate-to-edge: 5%;
+@circle-bg-translate-to-center:50%;
+@circle-bg-translate-to-center-asymmetric:35%;
+
+.circle-bg {
+  position: relative;
+  z-index: 1;
```

Comment:
I managed to remove z-index:1 from the .circle-bg
The solution when we have a section with its own background is to add the .low-z-index class to that section in order to give a lower z-index and keep it as a background. 
`.low-z-index{
  position: relative;
  z-index: -1;
}`